src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -408,6 +407,11 @@ export class HuiCardPicker extends LitElement {
           width: 100%;
           height: 100%;
           z-index: 1;
+          border: var(--ha-card-border-width, 1px) solid var(
+            --ha-card-border-color,
+            var(--divider-color)
+          );
+          border-radius: var(--ha-card-border-radius);
```

Comment:
Duplicate of line 356

Reply:
Nope, this is applying to the overlay, and 356 is to the card.

Reply:
Why is the border moved from the card to the overlay?

Reply:
Because it gets cropped off and weird otherwise.